engine/glfw/lib/android/android_init.c

```diff
@@ -723,18 +896,19 @@ int _glfwPlatformInit( void )
         return GL_FALSE;
     }
 
-    _glfwWin.display = EGL_NO_DISPLAY;
-    _glfwWin.context = EGL_NO_CONTEXT;
-    _glfwWin.surface = EGL_NO_SURFACE;
     _glfwWin.iconified = 1;
-    _glfwWin.paused = 0;
-    _glfwWin.app = g_AndroidApp;
 
-    int result = pipe(_glfwWin.m_Pipefd);
+    memset(&_glfwWinAndroid, 0, sizeof(_glfwWinAndroid));
+    _glfwWinAndroid.app = g_AndroidApp;
+    _glfwWinAndroid.display = EGL_NO_DISPLAY;
+    _glfwWinAndroid.context = EGL_NO_CONTEXT;
+    _glfwWinAndroid.surface = EGL_NO_SURFACE;
```

Comment:
Moved the android specific things from the _glfwnWin struct, in order to better control when and how the struct was cleared.